addon/components/pix-radio-button.hbs

```diff
@@ -0,0 +1,13 @@
+<label
+  for={{this.id}}
+  class="pix-radio-button {{if this.isDisabled ' pix-radio-button--disabled'}}">
+  <input
+    id={{this.id}}
```

Comment:
**question**
Niveau accessibilité je me souviens qu'il faut : 
- soit imbriquer notre élément dans la balise label
- soit mettre le `for=id`

Je me demande si mettre les deux ne fait pas doublons ?

Reply:
tout à fait, oubli de ma part quand j'ai décidé de l'imbriquer :)